scalafix-cli/src/main/scala/scalafix/cli/CliRunner.scala

```diff
@@ -77,9 +82,11 @@ case class CliRunner(
     display.stop()
     exitCode.get()
   }
+  private val sourceroot: AbsolutePath =
+    database.map(_.sourceroot).getOrElse(common.workingPath)
 
   def safeHandleInput(input: Input): ExitStatus = {
-    def path = input.path(common.workingPath)
+    def path = input.path(sourceroot)
```

Comment:
Review @stuhood This line makes the relative filename in the Mirror absolute according to --sourceroot.